packages/osc-ui/src/components/TextArea/TextArea.spec.tsx

```diff
@@ -18,27 +18,39 @@ test('should render an asterisk when input field is required', () => {
 
 test('should render an icon when passed as a prop', () => {
     render(
-        <TextArea
-            icon={{
-                content: <ExclamationTriangleIcon />,
-                label: 'Exclamation Triangle Icon',
-                type: 'error',
-            }}
-            id="enquiry"
-            name="Enquiry"
-            required={true}
-            wasSubmitted={true}
-        />
+        <SpritesheetProvider>
+            <TextArea
+                icon={{
+                    content: <Icon id="exclamation-mark" />,
+                    label: 'Exclamation Triangle Icon',
+                    type: 'error',
+                }}
+                id="enquiry"
+                name="Enquiry"
+                required={true}
+                wasSubmitted={true}
+            />
+        </SpritesheetProvider>
     );
-    expect(screen.getByText('Exclamation Triangle Icon')).toBeInTheDocument();
+
+    expect(document.querySelector('use')).toBeInTheDocument();
+    expect(document.querySelector('use')).toHaveAttribute('href');
```

Comment:
Rather than testing the existence of just the href, I would test that the href value is what you'd expect. Something like this:
```suggestion
    expect(document.querySelector('use')).toHaveAttribute('href', ./spritesheet.svg#exclamation-mark'');
```
Basically just asserting that the icon is what you'd expect

Reply:
Ah nice. I did actually try doing this but was getting some weird errors, didn't realise they needed to be comma separated. Updated 👍

Reply:
Nice! Yeah that comma is a bit of a gotcha 😅